wandb/sdk/artifacts/artifact.py

```diff
@@ -234,24 +234,32 @@ def _from_name(
         name: str,
         client: RetryingClient,
         organization: str = "",
+        enable_tracking: bool = False,
     ) -> Artifact:
+        server_supports_enabling_artifact_usage_tracking = (
+            InternalApi().server_project_type_introspection()
+        )
+        query_vars = ["$entityName: String!", "$projectName: String!", "$name: String!"]
+        query_args = ["name: $name"]
+        if server_supports_enabling_artifact_usage_tracking:
+            query_vars = [*query_vars, "$enableTracking: Boolean"]
+            query_args = [*query_args, "enableTracking: $enableTracking"]
+
+        vars_str = ", ".join(query_vars)
+        args_str = ", ".join(query_args)
+
         query = gql(
-            """
-            query ArtifactByName(
-                $entityName: String!,
-                $projectName: String!,
-                $name: String!
-            ) {
-                project(name: $projectName, entityName: $entityName) {
-                    artifact(name: $name) {
+            f"""
+            query ArtifactByName({vars_str}) {{
+                project(name: $projectName, entityName: $entityName) {{
+                    artifact({args_str}) {{
                         ...ArtifactFragment
-                    }
-                }
-            }
+                    }}
+                }}
+            }}
+            {cls._get_gql_artifact_fragment()}
```

Comment:
Sorry my bad, I just realized -- I auto-merged #8575, which pulled this method out as a module-level function (and broke this line of code).

quick fix should be:

```suggestion
            {_get_gql_artifact_fragment()}
```

Reply:
Thank you for pointing that out! I totally missed it when I "resolved" the merge conflict.